src/app/admin/components/filter-search/filter-search.component.html

```diff
@@ -0,0 +1,69 @@
+
+  <div class="list">
```

Comment:
I started following BEM in the Admin module. Change the selectors to follow BEM for consistency. 
Docs: http://getbem.com/